openapi.yml

```diff
@@ -62,6 +62,68 @@ paths:
           description: FlowMod messages sent.
         '400':
            description: Invalid JSON flow.
+  '/api/kytos/flow_manager/v2/flows_by_switch':
+    post:
+      tags:
+        - Add
+      summary: Insert flows by switch.
+      parameters:
+        - name: force
+          schema:
+            type: string
+          description: Option to ignore switch connection errors.
+          required: false
+          example: "true"
+      requestBody:
+        description: Flows by the especified switch.
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                switch_id:
```

Comment:
Looks like OpenAPI supports specifying a dynamic key that follows a regex pattern, can you narrow this down to a dpid regex? Also, we might need `additionalProperties: true`

```
      type: object
      properties:
        switch_id:
          type: string
          pattern: ^[TODO-dpid-regex]$
```

Reply:
This won't impact much at the moment since flow_manager isn't fully validating based on OpenAPI spec yet, but let's make sure it won't become a tech debt when it comes time to using the OpenAPI spec validation in the future

Reply:
Looks like pattern for properties were added to 3.1.0 with the addition of `patternProperties`. This is not available in 3.0.0 `flow_manager` uses. Do I update it?

Reply:
It might be worth updating to 3.1.0, `openapi-core` claims to support both, so in theory when the time comes in the future to also have validations with openapi-core it's aligned with that future plan.